misc-utils/fincore.c

```diff
@@ -42,40 +43,93 @@
    e.g. 128MB on x86_64. ( = N_PAGES_IN_WINDOW * 4096 ). */
 #define N_PAGES_IN_WINDOW ((size_t)(32 * 1024))
 
+#ifndef HAVE_CACHESTAT
+
+#include <linux/types.h>
+
+#ifndef __NR_cachestat
+#define __NR_cachestat 451
+#endif
+
```

Comment:
See configure script, it can define SYS_cachestat for you by UL_CHECK_SYSCALL([cachestat])